index.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+//1. Задача про ітерацію
+function iterativeOddSumTo(number) {
+
+    let sum = 0;
+
+    for (let i = 1; i <= number; i++) {
+        if (i % 2 !== 0) {
```

Comment:
```suggestion
        if (i % 2) {
```
i % 2 нам поверне або 1 або 0, все що не 0 зведеться до true, тому можна скоротити